<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMPrincipal</class>
 <widget class="QMainWindow" name="FMPrincipal">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>662</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FMPrincipal</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnNewSale">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Nova venda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>42</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Imagens/OpenedLock.png</normaloff>:/Imagens/OpenedLock.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lblCollabName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>520</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Usuário não logado</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGest_o">
    <property name="title">
     <string>Gestão</string>
    </property>
    <addaction name="actionStock"/>
    <addaction name="actionCollaborators"/>
    <addaction name="actionSales"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSistema">
    <property name="title">
     <string>Sistema</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuGest_o"/>
   <addaction name="menuSistema"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStock"/>
   <addaction name="actionCollaborators"/>
   <addaction name="actionSales"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionStock">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Imagens/stock.png</normaloff>:/Imagens/stock.png</iconset>
   </property>
   <property name="text">
    <string>Estoque</string>
   </property>
  </action>
  <action name="actionCollaborators">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Imagens/collaborator.png</normaloff>:/Imagens/collaborator.png</iconset>
   </property>
   <property name="text">
    <string>Colaboradores</string>
   </property>
  </action>
  <action name="actionSales">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Imagens/sell.png</normaloff>:/Imagens/sell.png</iconset>
   </property>
   <property name="text">
    <string>Vendas</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Imagens/exit.png</normaloff>:/Imagens/exit.png</iconset>
   </property>
   <property name="text">
    <string>SAIR</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Imagens/about.png</normaloff>:/Imagens/about.png</iconset>
   </property>
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
